.babelrc.js

```diff
@@ -6,10 +6,10 @@ if (process.env.NODE_ENV !== `test`) {
   ignore.push(`**/__tests__`)
 }
 
-const presetAbsPath = require(`path`).join(__dirname, '.babel-preset.js')
+//const presetAbsPath = require(`path`).join(__dirname, '.babel-preset.js')
```

Comment:
I am not 100% convinced I like commented code. Do you mind deleting it, or even better yet: Using the new value for the constant (see line 13)

Reply:
@mariobodemann i'm waiting for one of the JS folks to review this...this was there code so i'm not sure why it was added in the first place. for all i know it's suppposed to be there for some reason, but it caused the build to break